src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -2455,6 +2466,17 @@ protected void runMayThrow() throws Exception
         abstract protected void runMayThrow() throws Exception;
     }
 
+    public static void logRequestException(Exception exception, Collection<? extends ReadCommand> commands)
+    {
+        NoSpamLogger.log(logger, NoSpamLogger.Level.INFO, FAILURE_LOGGING_INTERVAL_SECONDS, TimeUnit.SECONDS,
```

Comment:
Can we get a flag to allow disabling this by default?  Would be nice to make it a hot-prop so we can disable if it becomes a problem